testsuite/integration-arquillian/tests/base/pom.xml

```diff
@@ -135,6 +136,12 @@
             <artifactId>jandex</artifactId>
             <version>2.1.3.Final</version>
         </dependency>
+        <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
+            <artifactId>system-rules</artifactId>
+            <version>${systemrules.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Where and why exactly is this used?

Reply:
This is used to manage env vars when running the test.